adobe-dmeservices/user-sync.py
# Refactor sign configs - move create and deactivate keys to connector

## Summary
The config keys to create and deactivate users on Neptune consoles have now been moved to the connector config file rather than the main config file. Schema validation and tests have been updated appropriately. Logic in the Sign Engine to call the corresponding methods to create or deactivate users has been changed to depend on the corresponding variable in the Connector object (i.e., if sign_connector.create_users, and so on) rather than the engine options.

I also removed all references to Neptune console per our conversations in Sprint meetings, with the exception of the comments in the config file.


## Testing Steps
No unit test - I believe those will be built upon in future stories. I did run a quick integration test to confirm syncing still works, and if create_users is enabled when connecting to a Neptune console, that request is completed successfully. We could probably do with a better warning message for when the create/deactivate flag is True on GPS - this presumably will return an error from that API call and right now I don't think that messaging goes out like we would want.



Fixes #xxx
